<launch>

    <node pkg="nodelet" type="nodelet" name="pcl_manager" args="manager" output="screen" />

    <node pkg="nodelet" type="nodelet" name="voxel_grid_master" args="load pcl/VoxelGrid pcl_manager" output="screen">
      <remap from="~input" to="/points2" />
      <rosparam>
        filter_field_name: z
        filter_limit_min: 0.0
        filter_limit_max: 3.5
        filter_limit_negative: False
        leaf_size: 0.008
      </rosparam>
    </node>


    <node pkg="tuni_whitegoods_perception" name="points_depthmap" type="depth_interface_node" output="screen"/>

    <node pkg="tuni_whitegoods_perception" name="pcl_fuse"  type="pcl_fusion_node" output="screen"/>

</launch>



